programs/compressed-pda/src/instructions.rs

```diff
@@ -206,6 +206,55 @@ pub struct InstructionDataTransfer {
     pub is_compress: bool,
 }
 
+pub struct NewAddressParams {
```

Comment:
Where is it being used?

Reply:
instead of the [address_... inputs in the struct above](https://github.com/Lightprotocol/light-protocol/pull/581/files#diff-ea7ccc35ef6c753c2be62abd415665488d7b5f28da47212664d50f2d9d25f542L197-L199) asked for feedback in chat whether anyone has reservations if not will use this.

Reply:
Maybe this pr should have been draft.

Reply:
now it's used